SampleProjects/NetworkLib/scripts/install.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# if we don't have an Ethernet library, then get the standard one
```

Comment:
This seems like a bug in the core library, was it not able to fetch this by specifying a dependency in `.arduino-ci.yml`?

Reply:
The dependency in `.arduino-ci.yml` would bring in the [official library](https://www.arduino.cc/en/reference/ethernet) (code [here](https://github.com/arduino-libraries/Ethernet)?). But for the purposes of this test in Arduino Ci, we'd like to test against our [fork](https://github.com/Arduino-CI/Ethernet) of the Ethernet library, since it is our fork that has `arduino_ci` support.

Reply:
Unless I'm missing something, this behavior of "install a standard library if a local one does not exist" is also implemented in the core library:
https://github.com/Arduino-CI/arduino_ci/blob/master/exe/arduino_ci.rb#L170-L173

In other words, if you manually install your own ethernet library locally, then listing a library of the same name in `.arduino-ci.yml` will not cause the CI runner to overwrite it.  The check is simple directory existence: https://github.com/Arduino-CI/arduino_ci/blob/master/lib/arduino_ci/arduino_cmd.rb#L238-L240

So I think this entire script and the reference to it can be deleted

Reply:
The comment does not do a good job of explaining what I want, so I've revised it. 
When we are running the CI tests (on Travis, Appveyor, or GitHub Actions), we won't have the Ethernet library present at first. We don't want `arduino_ci` do install it because it will install the "standard" one and we want the "nonstandard" one (my bad on the comment). But, I am running the CI tests on my development machine, I don't want it replacing my existing (development) Ethernet library. So, unless `.arduino-ci.yml` can specify a source (like a `Gemfile`), then I need the custom script. At least I think so! Does that make more sense?

Reply:
While the purpose was to use our custom Ethernet library (the reference to the "standard" library was a mistake), for purposes of validating that `arduino_ci` has the required pieces to compile the custom library it is probably sufficient to compile against the standard library, so the above comment can be ignored.